### PR TITLE
#7 - CVE : removed unused dependency causing CVE warning.

### LINKED ISSUE #7: Fix CVE
## Problem 

Currently, the build is broken because of a detected CVE in the dependencies. We need to check the CVE and remove the dependency if possible.

### PR DESCRIPTION
**Pull request by pj-spoelders**
Closes #7 
Removed unused dependency

### CODE REVIEW

**Comment by jakobbraun** [comment]
Please add a note to the changelog

**Comment by jakobbraun** [comment]
And also increase the version number

File: doc/changes/changes_0.1.2.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Exasol Database fundamentals for Java 0.1.2, released 05.07.2021
```
Comment (jakobbraun) [inline doc/changes/changes_0.1.2.md]:
```suggestion
# Exasol Database fundamentals for Java 0.1.2, released 2021-07-06
```

File: doc/changes/changes_0.1.2.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Code name:
```
Comment (jakobbraun) [inline doc/changes/changes_0.1.2.md]: Add a name for the release

File: pom.xml [hunk 3]
```diff
@@ -105,16 +99,32 @@
...
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>verify</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <modules>
+                        <!-- add modules here: -->
```
Comment (jakobbraun) [inline pom.xml]:
```suggestion
```

**Review by jakobbraun — CHANGES_REQUESTED** [review]

File: doc/changes/changes_0.1.2.md (hunk 2 above)
Comment (pj-spoelders) [reply doc/changes/changes_0.1.2.md]: done

**Review by jakobbraun — APPROVED** [review]

**Pull request merged**